traitlets/tests/test_traitlets.py

```diff
@@ -1585,8 +1585,8 @@ def coerce(self, value):
         return value
 
     def test_invalid_args(self):
-        self.assertRaises(TypeError, Tuple, 5)
```

Comment:
Calling a constructor with the wrong arguments does normally raise TypeError. Why has this changed?